linux/sound.cpp

```diff
@@ -31,6 +31,13 @@ void CSound::OpenJack ( const bool  bNoAutoJackConnect,
                         const char* jackClientName )
 {
     jack_status_t JackStatus;
+    const char *serverName;
+
+    if ((serverName = getenv("JACK_DEFAULT_SERVER")) == NULL) {
```

Comment:
With the rather spacing-centric coding style of Jamulus, it should probably look like this :)

```suggestion
    if ( ( serverName = getenv( "JACK_DEFAULT_SERVER" ) ) == NULL ) {
```

Reply:
I added the changes and force-pushed a new version.